tests/e2e/feegrant/grpc.go

```diff
@@ -0,0 +1,203 @@
+package feegrant
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/testutil"
+)
+
+func (s *E2ETestSuite) TestGrantGRPCAllowance() {
```

Comment:
Can we make them non E2E? and put them here if they do not exist: https://github.com/cosmos/cosmos-sdk/blob/c8ab555174ef5b1f8557b80361d8b83754f62f6a/x/feegrant/keeper/grpc_query_test.go
It would be great to delete the E2E and improve existing grpc tests instead of adding new ones at a different place.

Reply:
I was thinking that those ones are mocked, but If we don't think is necessary for them to be in e2e I will delete them.

Reply:
I think we are already testing them in feegrant's `grpc_query_test.go`, but the only difference is we are using QueryRequests there and RPC URLs here